Tests/XcodeProjTests/Tests/Fixtures.swift

```diff
@@ -1,42 +1,42 @@
 import Foundation
 import PathKit
-import XcodeProj
+@testable import XcodeProj
 
 func fixturesPath() -> Path {
     Path(#file).parent().parent().parent().parent() + "Fixtures"
 }
 
-func synchronizedRootGroupsFixture() -> (Path, [String: Any]) {
```

Comment:
What about renaming all the functions in this fixture to end in `...Data()` instead of `...Dictionary()`?